ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs

```diff
@@ -101,7 +101,7 @@ private T InitWithBasicProperties<T>(ReflectionItem item) where T : ItemSDPModel
             rval.AssembliesWithMoniker = _store.UWPMode ? null : MonikerizeAssemblyStrings(item);
             rval.PackagesWithMoniker = _store.UWPMode ? null : MonikerizePackageStrings(item, _store.PkgInfoMapping);
             rval.AttributesWithMoniker = item.Attributes?.Where(att => att.Visible)
-                .Select(att => new VersionedString() { Value = att.TypeFullName, Monikers = att.Monikers?.ToHashSet() })
+                .Select(att => new VersionedString() { Value = att.TypeFullName, Monikers = att.Monikers?.ToHashSet(), PerLanguage = ConvertNamedPerLanguage(att.TypeFullName, item) })
```

Comment:
don't need to convert to Markdown here, it's just xref string.

Reply:
Yes, in the  "ConvertNamedPerLanguage" method, there is a parameter isXRef to control, and its default value is false.